### ISSUE TITLE
Support for string literals as description

### ISSUE THREAD

**Issue by SpaceHeroGuide** [99designs/gqlgen#86]
As of version 0.12.3 the default style of describing a type or field is to add a string literal instead of a comment.
https://github.com/graphql/graphql-js/blob/master/src/utilities/extendSchema.js#L44

### Expected Behaviour
Parsing a schema with string literals as descriptions and generating the Golang files.

### Actual Behavior
Crashing due to invalid syntax.

### Minimal graphql.schema and models to reproduce
```graphql
"This is a description."
schema {
    query: Query
}
```

Results in `unable to parse schema: graphql: syntax error: unexpected "\"This is a comment.\"", expecting Ident (line 1, column 1)`

**Comment by imiskolee** [comment]
maybe start with comment symbol is better. like

```go
//This is a description.
schema {
    query: Query
}
```

**Comment by SpaceHeroGuide** [comment]
gqlgen should stick to the official documentation.

The documentation states that descriptions should be written as string literals. Comments are not descriptions.

**Comment by imiskolee** [comment]
@SpaceHeroGuide  got it. thanks!

**Comment by vektah** [comment]
The blocker on this, and a few other issues is the parser we use for schema. 

Upstream issue https://github.com/graph-gophers/graphql-go/issues/196

The other alternative is to fork https://github.com/tmc/graphql and get it up to date. A parser generator would be nice way nicer to work with than a hand rolled parser.

**Comment by geertjanvdk** [comment]
A small concern (or maybe I missed the scope of the issue), the GraphQL specification states that only [comments starting with # are supported](http://facebook.github.io/graphql/October2016/#sec-Comments). This will remain right?

**Comment by vektah** [comment]
Fixed in 0.4.0
